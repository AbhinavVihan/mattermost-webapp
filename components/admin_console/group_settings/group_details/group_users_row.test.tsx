// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import {shallow} from 'enzyme';
import React from 'react';

import GroupUsersRow from 'components/admin_console/group_settings/group_details/group_users_row';
import {renderWithIntl} from 'tests/react_testing_utils';

describe('components/admin_console/group_settings/group_details/GroupUsersRow', () => {
    test('should match snapshot', () => {
        const wrapper = renderWithIntl(
            <GroupUsersRow
                username='test'
                displayName='Test display name'
                email='test@test.com'
                userId='xxxxxxxxxxxxxxxxxxxxxxxxxx'
                lastPictureUpdate={0}
            />,
        );
        expect(wrapper).toMatchSnapshot();
    });
});
