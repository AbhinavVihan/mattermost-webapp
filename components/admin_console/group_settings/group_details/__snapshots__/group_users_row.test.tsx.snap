// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/admin_console/group_settings/group_details/GroupUsersRow should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="group-users-row"
      >
        <img
          alt="test profile image"
          class="Avatar Avatar-lg"
          loading="lazy"
          src="/api/v4/users/xxxxxxxxxxxxxxxxxxxxxxxxxx/image"
          tabindex="0"
        />
        <div
          class="user-data"
        >
          <div
            class="name-row"
          >
            <span
              class="username"
            >
              @test
            </span>
            -
            <span
              class="display-name"
            >
              Test display name
            </span>
          </div>
          <div>
            <span
              class="email-label"
            >
              Email:
            </span>
            <span
              class="email"
            >
              test@test.com
            </span>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="group-users-row"
    >
      <img
        alt="test profile image"
        class="Avatar Avatar-lg"
        loading="lazy"
        src="/api/v4/users/xxxxxxxxxxxxxxxxxxxxxxxxxx/image"
        tabindex="0"
      />
      <div
        class="user-data"
      >
        <div
          class="name-row"
        >
          <span
            class="username"
          >
            @test
          </span>
          -
          <span
            class="display-name"
          >
            Test display name
          </span>
        </div>
        <div>
          <span
            class="email-label"
          >
            Email:
          </span>
          <span
            class="email"
          >
            test@test.com
          </span>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
